### DISCUSSION TITLE
Allow aliasing....

### DISCUSSION THREAD

**Discussion by MHacker9404** [coreybutler/nvm-windows#604] [Ideas & Feature Requests] [answered]
Would you consider allowing aliasing?  This would permit testing new updates much easier - thanks

**Comment by coreybutler** [comment, marked as answer]
Yes, and this is planned for "rt" (successor project). The current challenge, albeit relatively minor, is persisting alias names. While it can be done with the settings.txt file, there are more effective storage mechanisms. 1.1.8 is (tentatively) the last release of NVM4W before I'll recommend using "rt" instead, so I've been hesitant to add new functionality in a final version (this really should be a 2.0.0 thing).

**Reply by MHacker9404** [reply]
dude - you da man!  If you need/want some help - gimme a holler

**Reply by rluvaton** [reply]
@coreybutler thank you for all of your great work!

Do you have a link for the _rt_ project?

**Edit:** never mind, [this](https://github.com/coreybutler/nvm-windows/discussions/565#discussioncomment-309103) answered me
